cypress/support/command.ts

```diff
@@ -0,0 +1,24 @@
+import '@testing-library/cypress/add-commands'
+
+Cypress.Commands.add('login', (email?: string, password?: string) => {
+  cy.visit('/')
+
+  cy.get('[data-testid="input-email"] > .MuiInputBase-input')
+    .should('be.visible')
```

Comment:
should à supprimer. ce test est à faire dans le test de auth ou de la page login

Reply:
Le should visible est pour eviter une erreur commune de cypress qui essaye de taper un truc trop vite
https://github.com/ABC-TransitionBasCarbone/bilan-carbone/actions/runs/11344952186/job/31550777143
https://github.com/cypress-io/cypress/issues/9075